src/main/java/com/vinylteam/vinyl/security/TokenCacheConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.vinylteam.vinyl.security;
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.concurrent.TimeUnit;
+
+@Configuration
+public class TokenCacheConfig {
+
+    @Bean
+    public Caffeine caffeineConfig() {
+        return Caffeine
```

Comment:
Absolutely readable in one line. If you dont have opposite code convetion on the project ofc